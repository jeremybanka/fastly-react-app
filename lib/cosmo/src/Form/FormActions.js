/* @flow */

import * as React from "react";
import { Box } from "../";

type Props = {
  children: React.Node,
};

const FormActions = (props: Props) => (
  <Box borderTop="light" paddingTop="md" marginTop="md">
    {props.children}
  </Box>
);

export default FormActions;
