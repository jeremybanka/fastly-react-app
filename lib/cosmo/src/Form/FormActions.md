The `<Form.Actions />` component should be a direct child of `<Form />`.
