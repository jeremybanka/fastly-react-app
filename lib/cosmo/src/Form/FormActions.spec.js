/* @flow */

import React from "react";
import { render } from "enzyme";
import FormActions from "./FormActions";

describe("FormActions test suite", () => {
  it("renders the stored snapshot", () => {
    const component = <FormActions>Hi</FormActions>;
    expect(render(component)).toMatchSnapshot();
  });
});
