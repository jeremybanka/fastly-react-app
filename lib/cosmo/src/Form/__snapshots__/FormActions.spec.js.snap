// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormActions test suite renders the stored snapshot 1`] = `
<div
  class="Box-sc-1v2etov-0 gLqHcs"
>
  Hi
</div>
`;
